changes/16498.md

```diff
@@ -0,0 +1,11 @@
+# PR #16498 Decouple Staged_ledger_diff.{Stable.Latest.t, t}
```

Comment:
This should either be deleted or updated to reflect only the user-facing changes. Suggest
```
Update the Staged_ledger_diff representation to store its proofs on disk. This allows us to reduce RAM usage by removing these large blobs from memory while we're not actively using them.
```